modules/release-notes/pages/relnotes.adoc

```diff
@@ -5,6 +5,106 @@
 
 Take a look at xref:introduction:whats-new.adoc[What's New?] for a list of new features and improvements in this release.
 
+[#release-704]
+== Release 7.0.4 (May 2022)
+Couchbase Server 7.0.4 was released in May 2022.
+This maintenance release contains fixes to issues.
+
+[#fixed-issues-704]
+=== Fixed Issues
+
+This section highlights the notable issues fixed in this release.
+
+=== Query Service
+
+[#table_fixedissues_v704-queryservice,cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-48438[MB-48438^]
+| *Summary*: Include bucketType in pools/default/b/<bucket-name> REST API
+
+| https://issues.couchbase.com/browse/MB-51581[MB-51581^]
+| *Summary*: JOIN query not covered
+
+| https://issues.couchbase.com/browse/MB-51453[MB-51453^]
+| *Summary*: IN/NOT IN filters not using Hash for evaluation
+
+| https://issues.couchbase.com/browse/MB-51431[MB-51431^]
+| *Summary*: Avoid circular link in annotated value hierarchy
+
+| https://issues.couchbase.com/browse/MB-51447[MB-51447^]
+| *Summary*: select executing a function executing a select returns the same results
+
+| https://issues.couchbase.com/browse/MB-51739[MB-51739^]
+| *Summary*: LEFT JOIN breaks with between operator on non-existing attribute
+
+| https://issues.couchbase.com/browse/MB-51578[MB-51578^]
+| *Summary*: Error 4321 generated when OR clause has a ($parameter is null) arm
+
+|===
+
+=== Analytics Service
+
+[#table_fixedissues_v704-analyticsservice,cols="25,66"]
+|===
+|Issue | Description
+
+| https://issues.couchbase.com/browse/MB-51676[MB-51676^]
+| *Summary*: To investigate the crash in MB-49471
+
+| https://issues.couchbase.com/browse/MB-51483[MB-51483^]
+| *Summary*: Data is not completely ingested into datasets
+
+| https://issues.couchbase.com/browse/MB-51482[MB-51482^]
+| *Summary*: Node is not upgraded and stuck trying to register with CC
+
+| https://issues.couchbase.com/browse/MB-51488[MB-51488^]
+| *Summary*: Ignore byte order marks when parsing external data
+
+| https://issues.couchbase.com/browse/MB-51700[MB-51700^]
+| *Summary*: CBAS service keeps crashing when one of the CBAS nodes is failed over
+
+| https://issues.couchbase.com/browse/MB-51412[MB-51412^]
+| *Summary*:  Graceful failover + delta recovery + rebalance fails
```

Comment:
This was fixed in 6.6.5 and forward ported to 7.0.4. We didn't mention it in the 6.6.5 release notes and we should not add it now.

Reply:
ok